source/features/resolve-conflict.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'dom-chef';
+import CodeMirror from 'codemirror';
+import features from '../libs/features';
+
+interface Editor extends CodeMirror.Editor, CodeMirror.Doc {
+}
+interface EditorDOMElement extends HTMLDivElement {
+	CodeMirror: Editor;
+}
+
+function embeddedInit() {
+	class Resolver {
+		editor: Editor
+
+		conflictNumber = 1
+
+		constructor() {
+			const codeMirrorElement: EditorDOMElement = document.querySelector('.CodeMirror');
+			this.editor = codeMirrorElement.CodeMirror;
```

Comment:
For one-off usage I’d rather just use `(codeMirrorElement as any).CodeMirror` instead of creating an interface.